src/inference/include/ie/cpu/cpu_config.hpp

```diff
@@ -42,5 +42,28 @@ DECLARE_CPU_CONFIG_KEY(DENORMALS_OPTIMIZATION);
 
 DECLARE_CPU_CONFIG_KEY(SPARSE_WEIGHTS_DECOMPRESSION_RATE);
 
+/**
+ * @brief The name for defining processor type used for CPU inference.
+ *
+ *  - UNDEFINED:       Default setting. All processors can be used on one socket platform. And only processors of
+ *                     physical cores can be used on two socket platform.
+ *  - ALL_CORE:        All processors can be used. If hyper threading is enabled, both processor of one performance-core
+ *                     can be used.
+ *  - PHY_CORE_ONLY:   Only processors of physical cores can be used. If hyper threading is enabled, only one processor
+ *                     of one performance-core can be used.
+ *  - P_CORE_ONLY:     Only processors of performance-cores can be used. If hyper threading is enabled, both processor
+ *                     of one performance-core can be used.
+ *  - E_CORE_ONLY:     Only processors of efficient-cores can be used.
+ *  - PHY_P_CORE_ONLY: Only processors of physical performance-cores can be used. If hyper threading is enabled, only
+ *                     one processor of one performance-core can be used.
+ */
+DECLARE_CPU_CONFIG_KEY(PROCESSOR_TYPE);
+DECLARE_CPU_CONFIG_VALUE(UNDEFINED);
+DECLARE_CPU_CONFIG_VALUE(ALL_CORE);
+DECLARE_CPU_CONFIG_VALUE(PHY_CORE_ONLY);
+DECLARE_CPU_CONFIG_VALUE(P_CORE_ONLY);
+DECLARE_CPU_CONFIG_VALUE(E_CORE_ONLY);
+DECLARE_CPU_CONFIG_VALUE(PHY_P_CORE_ONLY);
```

Comment:
I suppose we don't need to extend old API 1.0

Reply:
AUTO plugin will use this new property. Now we only support OV 1.0 API between plugins.

Reply:
Can't AUTO use new properties and just convert them to map<string, string> before passing to lower level plugin? To me extending user interface to work-around internal api issue doesn't look good.

Reply:
@vladimir-paramuzov  Properties for lower level plugin have been transferred to string as 1.0 API before Core send them to AUTO. 

After 2.0 API enabled between Core and plugins, we can move to properties mapas you mentioned.